docs/source/orderer/ordering_service.md

```diff
@@ -205,6 +205,13 @@ implementation the node will be used in), check out [our documentation on deploy
   up and manage than Kafka-based ordering services, and their design allows
   different organizations to contribute nodes to a distributed ordering service.
 
+* **BFT** (New!)
```

Comment:
The new BFT option was already introduced in line 188 above. Could you merge the section at line 188 with this section?
Also it is good to use language like "New as of v3.0" instead of "New!" so that we don't have to change the docs again in the next version.